### ISSUE TITLE
Error loading runtime modules in YARN

### ISSUE THREAD

**Issue by sabbyanandan** [spring-cloud/spring-cloud-dataflow-ui#16]
As a user, I'm trying to load `Runtime Modules` tab; however, I'm getting an empty page and the browser console includes the following error.

```
XHR finished loading: GET "http://localhost:9393/runtime/modules?page=0&size=10".(anonymous function) @ angular.js:9734sendReq @ angular.js:9538serverRequest @ angular.js:9255processQueue @ angular.js:13075(anonymous function) @ angular.js:13091Scope.$eval @ angular.js:14291Scope.$digest @ angular.js:14107(anonymous function) @ angular.js:14329completeOutstandingRequest @ angular.js:4858(anonymous function) @ angular.js:5231
angular.js:11500 TypeError: Cannot read property 'moduleStatusResourceList' of undefined
    at containers.js:39
    at processQueue (angular.js:13075)
    at angular.js:13091
    at Scope.$eval (angular.js:14291)
    at Scope.$digest (angular.js:14107)
    at Scope.$apply (angular.js:14395)
    at done (angular.js:9569)
    at completeRequest (angular.js:9756)
    at XMLHttpRequest.requestLoaded (angular.js:9697)(anonymous function) @ angular.js:11500(anonymous function) @ angular.js:8479processQueue @ angular.js:13083(anonymous function) @ angular.js:13091Scope.$eval @ angular.js:14291Scope.$digest @ angular.js:14107Scope.$apply @ angular.js:14395done @ angular.js:9569completeRequest @ angular.js:9756requestLoaded @ angular.js:9697
```

I launched `dataflow:>runtime modules` from the shell and I get the expected output.

```
╔═══════════════════════╤═══════════╤═════════════════════════════╗
║Module Id / Instance Id│Unit Status│No. of Instances / Attributes║
╚═══════════════════════╧═══════════╧═════════════════════════════╝
```

**Comment by ghillert** [comment]
@ilayaperumalg This is most likely a backend issue (That was also the case for the CF-related issue)

**Comment by ilayaperumalg** [comment]
@sabbyanandan I don't see the issue with `runtime modules` and the UI. 

```
dataflow:>runtime modules
╔═══════════════════════╤═══════════╤═════════════════════════════╗
║Module Id / Instance Id│Unit Status│No. of Instances / Attributes║
╠═══════════════════════╪═══════════╪═════════════════════════════╣
║a1.log                 │ deployed  │              1              ║
╟┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┼┈┈┈┈┈┈┈┈┈┈┈┼┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈╢
║a1.log                 │ deployed  │                             ║
╟───────────────────────┼───────────┼─────────────────────────────╢
║a1.time                │ deployed  │              1              ║
╟┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┼┈┈┈┈┈┈┈┈┈┈┈┼┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈╢
║a1.time                │ deployed  │                             ║
╚═══════════════════════╧═══════════╧═════════════════════════════╝

```

and the UI looks good as well.

If you have logs from `dataflow-admin-yarn` then that would help.

**Comment by sabbyanandan** [comment]
Cool. Trying this on the latest.

**Comment by sabbyanandan** [comment]
@ilayaperumalg / @ghillert: I tried with the latest and I did observe the same exact error, but only momentarily. I couldn't find any relevant logs of this failure neither in AppMAster's `stdout` or `stderr` - they're both clean. This momentary situation occurs when the module is in the process bootstrapping. 

Since this doesn't interrupt any user behavior and the fact it eventually comes up and running successfully, I'll close this.
